src/freezing/freezing_main.py

```diff
@@ -91,3 +97,24 @@ def run_freezing(
 def read_frozen_csv():
     # new functionality
     pass
+
+# function ready for use
+def _add_last_frozen_column(
```

Comment:
Can this be run at line 59 on the updated snapshot for run_first_snapshot_of_results (we'll use it for updates in the apply freezing adds and amends process)